vhost_user_backend/src/lib.rs

```diff
@@ -424,6 +424,13 @@ impl VringWorker {
     }
 }
 
+impl Drop for VringWorker {
+    fn drop(&mut self) {
+        // Ignore the result because there is nothing we can do about it.
+        let _ = epoll::close(self.epoll_fd);
+    }
+}
```

Comment:
Same here, store a File in the structure instead of the RawFd directly. This way, no need for a special Drop handling.